anxcloud/struct_dns_zones.go

```diff
@@ -34,10 +25,50 @@ func flattenDnsZones(dnsZones []zone.Zone) []interface{} {
 			"is_editable":        zone.IsEditable,
 			"validation_level":   zone.ValidationLevel,
 			"deployment_level":   zone.DeploymentLevel,
-			"dns_servers":        dnsServers,
+			"dns_servers":        flattenDNSServers(zone.DNSServers),
 		}
 
 		zones = append(zones, m)
 	}
 	return zones
 }
+
+func expandDNSServers(p []interface{}) []clouddnsv1.DNSServer {
+	dnsServers := make([]clouddnsv1.DNSServer, 0, len(p))
+	if len(p) < 1 {
```

Comment:
I think we can strip this `if`, the loop below will just not do anything in that case and after that we already return